dex/src/main/scala/com/wavesplatform/dex/market/OrderBookActor.scala

```diff
@@ -113,14 +114,9 @@ class OrderBookActor(settings: Settings,
             case QueueEvent.Placed(limitOrder)        => onAddOrder(request, limitOrder)
             case QueueEvent.PlacedMarket(marketOrder) => onAddOrder(request, marketOrder)
             case x: QueueEvent.Canceled               => onCancelOrder(request, x.orderId)
-            case _: QueueEvent.OrderBookDeleted       =>
-              // DEX-681
-              // private val reason = new EOFException("Order book was deleted")
-              // reason.setStackTrace(Array.empty)
-              // private val wsCloseMessage = classic.Status.Failure(reason)
-              //
-              // wsState.wsConnections.foreach(_ ! wsCloseMessage)
-              // wsState = wsState.withoutSubscriptions
+            case _: QueueEvent.OrderBookDeleted =>
+              log.warn("Order book was deleted, closing all WebSocket connections...")
+              aggregatedRef ! AggregatedOrderBookActor.WsCommand.CloseAllWsSubscriptions(OrderBookStopped(assetPair))
```

Comment:
Probably we don't need a special message to stop child. See https://doc.akka.io/docs/akka/current/typed/guide/tutorial_1.html#the-actor-lifecycle and try to write something like:
```scala
// AggregatedOrderBookActor
.receiveSignal {
  // ...
  case (_, PostStop) => // here the code
}
```

Reply:
Done